frontend/src/metabase-types/api/activity.ts

```diff
@@ -45,11 +47,7 @@ export interface RecentCollectionItem extends BaseRecentItem {
   model: "collection" | "dashboard" | "card" | "dataset" | "metric";
   can_write: boolean;
   database_id?: DatabaseId; // for models and questions
-  parent_collection: {
-    id: number | null;
-    name: string;
-    authority_level?: "official" | null;
-  };
+  parent_collection: Pick<Collection, "id" | "name" | "authority_level">;
```

Comment:
I noticed that in practice we're using the same data types so I figured we should do this